### PR TITLE
Remove version checks + PSR2

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: oneandonehosting.php [hunk 1]
```diff
@@ -56,21 +59,10 @@ public function install()
...
-			{
-				$this->context->controller->addCSS($this->_path.'views/css/configure.css');
-
-				if (version_compare(_PS_VERSION_, '1.6', '<') == true)
-					$this->context->controller->addCSS($this->_path.'views/css/configure-nobootstrap.css');
-			}
-			else
-			{
-				$html = '<link rel="stylesheet" href="'.$this->_path.'views/css/configure.css" type="text/css" />';
-				$html .= '<link rel="stylesheet" href="'.$this->_path.'views/css/configure-nobootstrap.css" type="text/css" />';
```
Comment (Quetzacoalt91) [inline oneandonehosting.php:71]: Hmm ... I wrote this part for PrestaShop 1.4. Even with the backward compatibility included, using `addCSS()` and `addJS()` doesn't work. Did you test it ?

**Comment by ZiZuu-store** [comment]
No I don't .. but I think that it should .. and if it does not work with backward compatibility then the backward compatibility could have an issue ..

AFAIK it should work .. if not then something elsewhere is wrong .. IMHO

File: oneandonehosting.php (hunk 1 above)
Comment (ghost) [inline oneandonehosting.php:71]: No I don't .. but I think that it should .. and if it does not work with backward compatibility then the backward compatibility could have an issue ..

AFAIK it should work .. if not then something elsewhere is wrong .. IMHO

**Comment by Quetzacoalt91** [comment]
Hi @ZiZuu-store,

Thanks for your PR, but I won't merge it today because it does not introduce a new functionality of fixes any bug.
Even if you are right about the potential issues which should be fixed in the backward compatibility, I currently prefer to keep this trick. I hope you understand that I do not want to risk to break a component used by all the modules working on PrestaShop 1.4. :wink: 

Best regards

**Pull request closed without merging**